src/components/bottomSheetScrollable/createBottomSheetScrollableComponent.tsx

```diff
@@ -77,6 +80,18 @@ export function createBottomSheetScrollableComponent<T, P>(
     );
     //#endregion
 
+    //#region callbacks
+    const handleContentSizeChange = useStableCallback(
+      (contentWidth: number, contentHeight: number) => {
+        animatedContentHeight.value = contentHeight;
```

Comment:
Doesn't this mean that I can't have additional content inside my bottom sheet if I'm using a scrollable component? The height of the sheet will be set to the height of the scrollable component instead of using the height of all the content in the sheet..